src/sources/tripadvisor/pois/convert.rs

```diff
@@ -145,9 +146,27 @@ pub fn build_poi(property: Property, geofinder: &AdminGeoFinder) -> Result<(u32,
         name: poi_type_name,
     };
 
+    let opening_hours = property
+        .hours
+        .inner
+        .iter()
+        .filter(|day| day.time.is_some())
+        .map(|day| {
+            let opening_times: String = day
+                .time
+                .iter()
+                .flatten()
+                .map(|time| format!("{}-{}", time.open_time, time.close_time))
+                .join(",");
+            format!("{} {}", day.name.get(0..2).unwrap_or(""), opening_times)
+        })
+        .join(";");
```

Comment:
That's not very important but there is a space after rule separators [in the original specification](https://wiki.openstreetmap.org/wiki/Key:opening_hours/specification#normal_rule_separator), that's not an issue with current implementation of Idunn as it uses the JS library as a backend but it could be a pain if we change library someday.

```suggestion
        .join("; ");
```